package.json

```diff
@@ -28,5 +28,6 @@
     "eslint-plugin-react-refresh": "^0.4.3",
     "vite": "^4.4.5",
     "vite-plugin-eslint": "^1.8.1"
-  }
+  },
+  "packageManager": "npm@10.8.1+sha512.0e9d42e92bd2318408ed81eaff2da5f78baf23ee7d12a6eed44a6e2901d0f29d7ab715d1b918ade601f72e769a824d9a5c322383f22bbbda5dd396e79de2a077"
```

Comment:
I assume this is fine but teach me: does this lock the package manager version?

Reply:
Sort of, you can still use any package manager and any version you want, but if you use corepack (which is installed by default with node now, and should become de facto across most projects) then it will install and use the correct package manager for you.

Reply:
Merge time baby!